Foundation/include/Poco/NumberFormatter.h

```diff
@@ -43,6 +43,15 @@ class Foundation_API NumberFormatter
 		FMT_ON_OFF
 	};
 
+	enum class Options
```

Comment:
Also add comment here to explain what this means.

Reply:
Resolved in new commit.